File: cpp/ql/test/query-tests/jsf/4.24 Control Flow Structures/AV Rule 193/nested.c

void nested1(int i) {
    switch (i) {
        case 1:
            i = 1;
            break;
        {
            ; ; ; ; ; ; ;
            case 2:
                i = 2;
                break;
        }
        default:
            i = 3;
    }
}

void nested2(int i) {
    switch (i) {
        case 1:
            i = 1;
            break;
        {
            ; ; ; ; ; ; ;
            default:
                i = 3;
        }
        case 2:
            i = 2;
            break;
    }
}

File: rust/ql/test/library-tests/path-resolution/my/nested.rs
pub mod nested1 {
    pub mod nested2 {
        pub fn f() {
            println!("nested.rs:nested1::nested2::f"); // $ item=println
        } // I4

        fn g() {
            println!("nested.rs:nested1::nested2::g"); // $ item=println
            f(); // $ item=I4
        } // I5
    } // I3

    fn g() {
        println!("nested.rs:nested1::g"); // $ item=println
        nested2::f(); // $ item=I4
    } // I6
} // I1

pub fn g() {
    println!("nested.rs::g"); // $ item=println
    nested1::nested2::f(); // $ item=I4
} // I7
